examples/platform/silabs/BaseApplication.cpp

```diff
@@ -242,7 +242,12 @@ CHIP_ERROR BaseApplication::Init()
     }
 
     PlatformMgr().AddEventHandler(OnPlatformEvent, 0);
+#ifdef SL_WIFI
+    sIsProvisioned = ConnectivityMgr().IsWiFiStationProvisioned();
+#endif /* SL_WIFI */
+#if CHIP_ENABLE_OPENTHREAD
     sIsProvisioned = ConnectivityMgr().IsThreadProvisioned();
+#endif
```

Comment:
Should it be and `if/else` instead of two back to back `if`?

Reply:
For now two if should be the way to go